source/process/overview.rst

```diff
@@ -0,0 +1,123 @@
+
+The design is successful if 100% of team members use Mattermost for internal communications, and are largely off of email and proprietary SaaS products that lock-in user data as part of their business model.
+
+See `Mattermost Manifesto <https://www.mattermost.org/manifesto/>`__ for more details.
+
+Community Systems
+-----------------
+
+Please see `Community Systems <http://docs.mattermost.com/process/community-systems.html>`__ documentation for how feedback from community constituents is processed into `tickets accepting pull requests. <http://docs.mattermost.com/process/accepting-pull-request.html>`__
+
+Tickets
+-------
+
+Mattermost priorities are managed in Jira tickets, which are created by the core team via feedback from community systems as well as through the planning processes.
+
+Triage
+~~~~~~
+
+On non-holiday weekdays new tickets are reviewed in a process called "triage", and assigned a Fix Version of "backlog", indicating the ticket has enough specificity that it can be assigned to a developer to be completed.
+
+By default, all tickets are created as public unless they contain sensitive information. The triage process reviews them for sufficient
+specificity. If the ticket is unclear, triage may reassign the ticket back to the original reporter to add more details.
+
+View `current issues scheduled for the next triage meeting <https://mattermost.atlassian.net/browse/PLT-1203?filter=10105>`__.
+
+Re-triage
+^^^^^^^^^
+
+If someone feels an existing ticket should be re-examined, they can add "triage" to the Fix Version and it will be routed to the triage team for review at the next meeting.
+
+Release Planning
+~~~~~~~~~~~~~~~~
+
+Release planning sets the "Fix Version" of tickets to one of the upcoming bi-monthly releases. The Fix Version is an estimate of when a
+feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint.
+
+Sprint Planning
+~~~~~~~~~~~~~~~
+
+Tickets to be completed in the upcoming two weeks are organized on Tuesdays, with input from developers.
+
+Pull Requests
+-------------
+
+Core Team Weekly Rhythm
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Core team work on tickets in the active sprint on a weekly basis, which flow into GitHub Pull Requests.
+
+Each Pull Request needs a minimum of two reviews by other core team developers before it is merged, with possible feedback shared as reviews happen.
+
+Key contributors might also pick up tickets, or through conversations with the core team contribute pull requests as needed.
+
+Community Contributions
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Community members following the `Contribution Guidelines <http://docs.mattermost.com/developer/contribution-guide.html#code-contribution-guidelines>`__ might also submit pull requests. Pull requests should not disable existing functionality without a Jira ticket, which are opened `via the feature ideas process <http://www.mattermost.org/feature-requests/>`__.
```

Comment:
I'm not sure what "disable existing functionality without a JIRA ticket" means? Probably needs updating to either has a JIRA or GitHub ticket and any contributions from the community need PM approval?